Marlin/Marlin_main.cpp

```diff
@@ -7040,9 +7064,11 @@ void process_next_command() {
           break;
       #endif // ENABLED(FILAMENT_WIDTH_SENSOR)
 
-      case 410: // M410 quickstop - Abort all the planned moves.
-        gcode_M410();
-        break;
+      #if DISABLED(EMERGENCY_PARSER)
+        case 410: // M410 quickstop - Abort all the planned moves.
+          gcode_M410();
+          break;
+      #endif
```

Comment:
Wouldn't this cause the commands to be returned with "unknown command" if they are not handled ? I'd suggest leaving the 'case' + 'break', but put the gcode_M410(); call inside the #if. Same for M112 and M108.